beta-src/src/utils/map/drawSupportMoveOrders.ts

```diff
@@ -10,6 +10,7 @@ export default function drawSupportMoveOrders(
   data: GameDataResponse["data"],
   maps: GameStateMaps,
   ordersMeta: OrdersMeta,
+  ownUnits: string[],
```

Comment:
Use `GameState["ownUnits"]` here that way if the type ever changes it will only need to be changed in one place.